### PR TITLE
doc + cleanup

### PR DESCRIPTION
**Pull request by ttossavainen**

### CODE REVIEW

File: Frends.AMQP.Send/Frends.AMQP.Send.Tests/TestAmqpBrokerHelper.cs [hunk 1]
```diff
@@ -6,133 +6,111 @@
...
 
-namespace Frends.AMQP.Send.Tests
+namespace Frends.AMQP.Send.Tests;
+
+/// These tests here use a test AMQP server. The test server is started and killed automatically.
+/// https://github.com/Azure/amqpnetlite/tree/master/test/TestAmqpBroker
+/// 
+/// If you want to test this against a real ActiveMQ server, Docker is recommended:
+/// docker pull rmohr/activemq
+/// docker run -p 5672:5672 -p 8161:8161 rmohr/activemq
```
Comment (Svenskapojkarna) [inline Frends.AMQP.Send/Frends.AMQP.Send.Tests/TestAmqpBrokerHelper.cs:16]: Please add `-d` to docker command to run the container in detached mode (in the background).

File: Frends.AMQP.Send/Frends.AMQP.Send.Tests/Frends.Amqp.Send.Tests.cs [hunk 2]
```diff
@@ -1,67 +1,64 @@
...
-            }
-            else
-            {
-                SecureBusAddress = externalAmqpsConnectionString;
-            }
-        }
+    [SetUp]
+    public void SetUp()
+    {
+        var externalAmqpsConnectionString = Environment.GetEnvironmentVariable("AMQPS_CONN_STR");
```
Comment (Svenskapojkarna) [inline Frends.AMQP.Send/Frends.AMQP.Send.Tests/Frends.Amqp.Send.Tests.cs:18]: We can use the docker container in unit tests, so the connection string can be hardcoded.

**Comment by Svenskapojkarna** [comment]
Please change the workflow in https://github.com/FrendsPlatform/Frends.AMQP/blob/main/.github/workflows/Send_test.yml and https://github.com/FrendsPlatform/Frends.AMQP/blob/main/.github/workflows/Send_main.yml to linux workflow and user docker container as test AMQP. You can remove the connection string env var since the connection string can be hardcoded to test.

File: Frends.AMQP.Send/Frends.AMQP.Send.Tests/Frends.Amqp.Send.Tests.cs (hunk 2 above)
Comment (ttossavainen) [reply Frends.AMQP.Send/Frends.AMQP.Send.Tests/Frends.Amqp.Send.Tests.cs:18]: Actually there is no need for this one when test AMQP server is up and running in docker. Removed.

File: Frends.AMQP.Send/Frends.AMQP.Send/Definitions/Enums.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+/// SearchCertificateBy options.
+/// </summary>
+public enum SearchCertificateBy
+{
+#pragma warning disable CS1591 // self explanatory
+    DontUseCertificate,
+    Issuer,
+    File
+#pragma warning restore CS1591 // self explanatory
+};
```
Comment (Svenskapojkarna) [inline Frends.AMQP.Send/Frends.AMQP.Send/Definitions/Enums.cs:13]: No need for semicolon here.

File: Frends.AMQP.Send/Frends.AMQP.Send/Frends.AMQP.Send.cs [hunk 4]
```diff
@@ -6,174 +6,143 @@
...
-        /// <param name="options">Defines additional properties of connection.</param>
-        /// <param name="messageProperties">Defines additional properties of messgae.</param>
-        /// <param name="cancellationToken"></param>
-        /// <returns>Object { bool Success }</returns>
-        public static async Task<SendMessageResult> Send([PropertyTab] InputSender input, [PropertyTab] Options options, [PropertyTab] AmqpMessageProperties messageProperties, CancellationToken cancellationToken)
+        var conn = await CreateConnection(input.BusUri, options.SearchClientCertificateBy, options.DisableServerCertValidation, options.Issuer, options.PfxFilePath, options.PfxPassword);
+        var session = new Session(conn);
+        var sender = new SenderLink(session, options.LinkName, input.QueueOrTopicName);
+
+        cancellationToken.ThrowIfCancellationRequested();
```
Comment (Svenskapojkarna) [inline Frends.AMQP.Send/Frends.AMQP.Send/Frends.AMQP.Send.cs:35]: No need to check cancellation here.

File: Frends.AMQP.Send/Frends.AMQP.Send/Frends.AMQP.Send.csproj [hunk 5]
```diff
@@ -12,7 +12,7 @@
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
```
Comment (Svenskapojkarna) [inline Frends.AMQP.Send/Frends.AMQP.Send/Frends.AMQP.Send.csproj:12]: https://github.com/FrendsPlatform/Frends.AMQP/blob/89f37a0d6fb3c785fa6134400195ef76edf0146b/Frends.AMQP.Send/Frends.AMQP.Send/Frends.AMQP.Send.csproj#L5
Please update version.

**Review by Svenskapojkarna — APPROVED** [review]

**Pull request merged**